modules/nf-core/kraken2/add/meta.yml

```
@@ -0,0 +1,55 @@
+name: kraken2_add
+description: Adds fasta files to a Kraken2 taxonomic database
+keywords:
+  - metagenomics
+  - db
+  - classification
+  - build
+tools:
+  - kraken2:
+      description: "Kraken2 is a system for assigning taxonomic labels to short DNA sequences, usually obtained through metagenomic studies."
+      homepage: https://ccb.jhu.edu/software/kraken2/
+      documentation: https://github.com/DerrickWood/kraken2/wiki/Manual
+      tool_dev_url: "https://github.com/DerrickWood/kraken2"
+      doi: 10.1186/s13059-019-1891-0
+      licence: ["MIT"]
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - fasta:
+      type: file
+      description: fasta file that will be added to the database
+      pattern: "*.{fa,fasta,fna,ffn}"
+  - taxonomy_names:
+      type: file
+      description: used for associating sequences with taxonomy IDs
+      pattern: "*.dmp"
+  - taxonomy_nodes:
+      type: file
+      description: tree nodes using NCBI taxonomy nomenclature
+      pattern: "*.dmp"
+  - accession2taxid:
+      type: file
+      description: associates sequence accession IDs to taxonomy IDs
+      pattern: "*.accession2taxid"
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false ]`
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - db:
+      type: directory
+      description: contains required files to build the database
+      pattern: "*/"
```

```suggestion
  - db:
      type: directory
      description: contains required files to build the database
      pattern: "*/"
  - versions:
      type: file
      description: File containing software versions
      pattern: "versions.yml"
```
Rearranged since the meta and db are part of the same output channel